docs/update-media-paths.md

```diff
@@ -0,0 +1,40 @@
+# TubeSync
+
+## Advanced usage guide - update media paths from the command line
+
+This command updates all media paths with what is current in the database.
+This may want to be done when the Media Format has been changed for a source,
+especially if the folder structure has changed.
+
+
+## Requirements
+
+At least one source and media is present
```

Comment:
Shouldn't the code for this command just warn the user when there are no sources or no downloaded media items?